QueryDispatch/Listeners/PatientEventListener.cs

```diff
@@ -61,29 +78,26 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
                         }
                         catch (ConsumeException e)
                         {
-                            _logger.LogError($"Consume failure, potentially schema related: {e.Error.Reason}");
-                            var potentialFacilityId = Encoding.UTF8.GetString(e.ConsumerRecord.Message.Key);
-
-                            var auditValue = new AuditEventMessage
+                            var facilityId = Encoding.UTF8.GetString(e.ConsumerRecord.Message.Key);
+                            var converted_record = new ConsumeResult<string, string>()
                             {
-                                FacilityId = potentialFacilityId,
-                                Action = AuditEventType.Query,
-                                ServiceName = "QueryDispatch",
-                                EventDate = DateTime.UtcNow,
-                                Notes = $"Kafka PatientEvent consume failure, potentially schema related \nException Message: {e.Error}",
+                                Message = new Message<string, string>()
+                                {
+                                    Key = facilityId,
+                                    Value = Encoding.UTF8.GetString(e.ConsumerRecord.Message.Value),
+                                    Headers = e.ConsumerRecord.Message.Headers
+                                }
```

Comment:
Probably want some null checking around the message key and value.  A null `e.ConsumerRecord.Message.Key`/`Value` here will cause `Encoding.UTF8.GetString` to throw a `NullReferenceException` and bring down the service.